tests/unit/badge/badge.renderBadge.spec.js

```diff
@@ -0,0 +1,42 @@
+import { shallow } from 'enzyme';
```

Comment:
1. If we talking about single responsibility and unit testing then you need to mock any call from `render` to `renderBadge`. Another point: would be nice to have files per public (part of the contract) function, not just a private helpers (function called only inside the file). You can easily replace that function call with a local variable in `render` function and reduce the number of files and function calls.
2. You have 2 the same snapshots
3. How would you reduce conflicts if your snapshots produced by one part of the code? I'd say you will increase it;)
4. Yep, you *just* need to select **correct** file
5. But we don't have it here, would be nice to follow pattern of current package?